# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../pam/pam-1.1.5-glibc-2.16-RLIMIT_NOFILE.patch
# Copyright (C) 2012 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

Fix to build with glibc >= 2.16

--- Linux-PAM-1.1.5/modules/pam_unix/pam_unix_acct.c.orig	2012-08-05 05:39:45.473424157 +0200
+++ Linux-PAM-1.1.5/modules/pam_unix/pam_unix_acct.c	2012-08-05 05:46:48.307995856 +0200
@@ -41,6 +41,7 @@
 #include <string.h>
 #include <unistd.h>
 #include <sys/types.h>
+#include <sys/resource.h>	/* RLIMIT_NOFILE */
 #include <syslog.h>
 #include <pwd.h>
 #include <shadow.h>
